src/components/common/EthHashInfo/styles.module.css

```diff
@@ -1,41 +1,20 @@
 .container {
-  display: flex;
-  align-items: center;
-  gap: 0.5em;
-  line-height: 1.4;
-}
-
-.avatar {
-  flex-shrink: 0;
+  position: relative;
 }
 
-.resizeAvatar,
-.resizeAvatar * {
-  width: 2.3em !important;
-  height: 2.3em !important;
-}
-
-.addressRow {
+.emojiWrapper {
+  text-align: center;
   display: flex;
-  align-items: center;
-  gap: 0.25em;
-  white-space: nowrap;
-}
-
-.nameRow {
-  overflow: hidden;
-}
-
-.mobileAddress {
-  display: none;
-}
-
-@media (max-width: 599.95px) {
-  .mobileAddress {
-    display: inline-block;
-  }
-
-  .desktopAddress {
-    display: none;
-  }
+  flex-direction: column;
+  justify-content: center;
+  position: absolute;
+  z-index: 1;
+  top: 0;
```

Comment:
For small identicons its not aligned correctly. We could add `top: 50%; transform: translateY(-50%)` to mitigate that.
<img width="75" alt="Screenshot 2023-08-01 at 14 58 20" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/55058f2e-e517-4517-b99a-34bb13778118">

Reply:
It didn't help, I disabled them for avatars < 20px.